PrivateHeaders/Ably/ARTChannel.h

```diff
@@ -0,0 +1 @@
+../.././Source/ARTChannel.h
```

Comment:
Thanks for the header changes. However, ARTChannel, ARTChannels, ARTDataQuery, ARTLog, ARTStatus, ARTTypes, ARTRealtimeChannels are not listed in the private module in the modulemap, so they **should** be in `include/` and exposed to users. All the headers which are in the private module should **not** be in the `include/`, but those that aren't should be in the `include/`. 

Also, you've missed more headers in the `include/` so users can't use features. e.g. `ARTRealtime.h` is not exposed, so users cannot subscribe to a channel. I haven't checked other headers.

You can have a look at my branch, which is based off the existing `Ably.modulemap`:
All the public headers [here](https://github.com/ably/ably-cocoa/tree/feature/ben-butterworth-temporary-spm/Sources/Ably/include)

All the internet headers [here](https://github.com/ably/ably-cocoa/tree/feature/ben-butterworth-temporary-spm/Sources/Ably/Internal)

Reply:
Looks like you've fixed it